### ISSUE TITLE
Website still using webfaction's shared certificate

### ISSUE THREAD

**Issue by Diginized** [will-in-wi/letsencrypt-webfaction#93]
Hello, 

Thank you for creating this project, I'm very excited to get this up and running.  I was starting to write this post with a question in it, but I waited an hour and after a few double-checks I eventually stumbled into my answer.  I thought I'd offer this for others who may run into the same issue, or in case you wanted to update the documentation. 

I successfully finished all of the steps in the instructions, and the domain's public folder was successfully showing the .well-known/acme-challenge/ subfolder with two files in it, which I think means it worked.  Likewise, when I sign into the webfaction control panel, I do see a new cert under the Domains > SSL Certificates tab of type "manually added", so I assume this means the fetch from LetsEncrypt and the API push into webfaction worked. 

However, when I visited the domain, it appeared to still be referencing webfaction's wildcard cert and provides the normal browser warning: 
```
NET::ERR_CERT_COMMON_NAME_INVALID
Subject: *.webfaction.com
Issuer: Trustwave Organization Validation SHA256 CA, Level 1
Expires on: Jan 16, 2020
```

Eventually I realized that if I enter the Webfaction Control Panel > Websites and scroll down to my domain, I need to change its settings to HTTPS, but what I didn't know before is that this step needs to happen **after** the cert has been manually added using letsencrypt_webfaction.  I'd done it before adding the cert.  After the cert is present, the HTTP/HTTPS toggle comes with an additional dropdown menu that wasn't there before, and in this dropdown you can select whether you want to use the default Webfaction shared cert, or one of your manually added certs.  Selecting the manually added one completes the setup and everything works. 

The existing letsencrypt_webfaction docs link to the Webfaction docs for how to automatically redirect http to https, but they don't mention that this step needs to happen first.   It would be nice to have mention of this in the **Operation** section of the docs. 

Thanks again for your work creating this great tool ! 
-Diginized

**Comment by will-in-wi** [comment]
Good point. I'll update the docs on my next round of updates.

One note, if you don't have an http version of your site, even if all it does is redirect to https, renewing the cert in 90 days will fail.
